changelog.d/15121.misc

```diff
@@ -0,0 +1 @@
+Route remote key requests via federation senders.
```

Comment:
(As I've left this comment on #15133 and #15134 I should just copy it here...)

I think we also need to update:

* `docker/configure_workers_and_start.py` so complement uses this
* https://github.com/matrix-org/sytest/blob/d1c931af1179848526d323f3955a9f21f81d600f/lib/SyTest/Homeserver/Synapse.pm#L1257 so sytest uses this.
* `docs/workers.md`

Reply:
The upgrade notes will need amending too; I think federation senders will need to be declared in the `instance_map`.